dspace-api/src/main/java/org/dspace/content/packager/PDFPackager.java

```diff
@@ -299,7 +302,17 @@ private void crosswalkPDF(Context context, Item item, InputStream metadata)
 
         try
         {
-            PDFParser parser = new PDFParser(metadata);
+            ScratchFile scratchFile = null;
+            try
+            {
+                scratchFile = new ScratchFile(MemoryUsageSetting.setupMixed(104857600)); // use up to 100 MB memory, fallback to temp file (unlimited size)
```

Comment:
@KevinVdV I tried to avoid adding yet another configuration option. What do you think of this solution? Keep in mind that this runs from CLI, where the -Xmx is currently 256 MB.